crates/node/src/chain_spec.rs

```diff
@@ -56,6 +61,9 @@ pub fn development_config(enable_manual_seal: Option<bool>) -> Result<DevChainSp
         "dev",
         ChainType::Development,
         move || {
+            // Logging the development account
```

Comment:
Read it only once.
There. 

Then pass it by ref as an argument to `print_development_accounts` and ` testnet_genesis` methods.

This way you still only read it once, and it don't have to be `static`

Reply:
You are right. I was thinking I should avoid changing the signature of `testnet_genesis`, but it was a stupid take on my part.

Reply:
BTW, using only references everywhere for this didn't work, so I had to resort to some cloning anyway.

Reply:
What was the problem with references?

Reply:
Replied below